lib/ansible/cli/doc.py

```diff
@@ -283,6 +283,10 @@ def find_plugins(path, ptype):
             display.vvvv("%s does not exist" % path)
             return plugin_list
 
+        if not os.path.isdir(path):
```

Comment:
I did this because the below `os.listdir` failed on https://github.com/ansible/ansible/blob/devel/test/runner/lib/ansible_util.py#L48 in a testing environment. Figured it's a good check to have anyway?